libs/SalesforceSDKCore/SalesforceSDKCoreTests/ScreenLockManagerTests.swift

```diff
@@ -50,37 +49,37 @@ class ScreenLockManagerTests: XCTestCase {
         ScreenLockManagerInternal.shared.handleAppForeground()
         XCTAssertTrue(SFSDKWindowManager.shared().screenLockWindow().isEnabled())
         wait(for: [expectation], timeout: 1)
-        
+//        
         // Unlock
         SFSDKWindowManager.shared().screenLockWindow().dismissWindow()
```

Comment:
So the tests run fine locally.  However, when run on CI, this line is causing an exception to be thrown.  I had to basically do a bit of a committing bisect where I’d comment out some code and comment the other half in to narrow down where the failure seems to be happening. When you log at the CI logs it doesn't give a lot of informaiton as what is happening.